bazel/crc32c.BUILD

```diff
@@ -49,70 +52,79 @@ crc32c_SRCS = [
     "src/crc32c.cc",
 ]
 
-config_setting(
-    name = "windows",
-    constraint_values = [
-        "@platforms//os:windows",
+# select() cannot be use recursively, need to introduce temporary conjunctions.
+selects.config_setting_group(
+    name = "linux_sse42",
+    match_all = [
+        "@platforms//os:linux",
         "@platforms//cpu:x86_64",
     ],
-    visibility = ["//visibility:public"],
 )
 
-config_setting(
-    name = "linux_x86_64",
-    constraint_values = [
-        "@platforms//os:linux",
+selects.config_setting_group(
+    name = "macos_sse42",
+    match_all = [
+        "@platforms//os:macos",
         "@platforms//cpu:x86_64",
     ],
-    visibility = ["//visibility:public"],
 )
 
-config_setting(
-    name = "macos",
-    constraint_values = [
-        "@platforms//os:macos",
+selects.config_setting_group(
+    name = "windows_sse42",
+    match_all = [
+        "@platforms//os:windows",
+        "@platforms//cpu:x86_64",
     ],
-    visibility = ["//visibility:public"],
 )
 
-sse42_copts = select({
-    ":windows": ["/arch:AVX"],
-    ":linux_x86_64": ["-msse4.2"],
-    ":macos": ["-msse4.2"],
+crc32c_copts = select({
+    ":linux_sse42": ["-msse4.2"],
+    ":macos_sse42": ["-msse4.2"],
+    ":windows_sse42": ["/arch:AVX"],
+    # No special flags are needed for ARM64+CRC32C.
     "//conditions:default": [],
 })
 
-sse42_enabled = select({
-    ":windows": "1",
-    ":linux_x86_64": "1",
-    ":macos": "1",
-    "//conditions:default": "0",
-})
-
-load(":configure_template.bzl", "configure_template")
-
 configure_template(
     name = "generate_config",
     output = "crc32c/crc32c_config.h",
     substitutions = {
         "#cmakedefine01": "#define",
+        # On x86_64 endianness is always little end.  ARM64 is bi-endian,
```

Comment:
s/end/endian/?

Or just "x86_64 is always little endian."

Reply:
Done.